recipes/clame/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+echo "SDSL compilation"
+export C_INCLUDE_PATH=${PREFIX}/include
+export LIBRARY_PATH=${PREFIX}/lib
+
+git clone https://github.com/simongog/sdsl-lite.git
```

Comment:
This library is available in conda-forge. 

https://anaconda.org/conda-forge/sdsl-lite
https://github.com/conda-forge/sdsl-lite-feedstock

Is it necessary to build it from source here? This seems to build directly from the `master` branch, which will be a moving target.

Reply:
I added sdsl-lite recipe into meta.yaml build requirements. What is the right path to be included into the Makefile?. I tried with -I $PREFIX/include but it doesn't work.
Thanks